### PR TITLE
Add list privacy information to mailman-API

### PR DESCRIPTION
**Pull request by Darksshades**
You can pass a ?private=True to receive the list privacy settings, if it's a private list or not.

### CODE REVIEW

**Comment by Darksshades** [comment]
@seocam, modifications to mailman-api to get the privacy settings of the mailinglists

File: mailmanapi/api.py [hunk 1]
```diff
@@ -32,16 +32,23 @@ def list_lists():
...
-                lists.append((listname, mlist.description.decode('latin1')))
+                list_values.append(mlist.description.decode('latin1'))
+            if include_private:
+                list_values.append(bool(mlist.archive_private))
+
+            if len(list_values) == 1:
+                lists.append(list_values[0])
             else:
-                lists.append(listname)
+                lists.append(list_values)
```
Comment (terceiro) [inline mailmanapi/api.py]: you are changing the structure of the response from a simple array to an array of arrays. Is that on purpose?

Comment (Darksshades) [inline mailmanapi/api.py]: We changed values from tuples to lists. The jsonify method at the end was already converting the response to an arrays of arrays even in the old version IF the description parameter was passed. And a single array if no parameter.
If no parameter is passed the now, line 49 will take care and send the response as a single array like always. If there are parameters, like the old version, it'll return a json with an array of arrays like [ [ 'listname', 'parameter1', '...' ], [...] ]

**Comment by seocam** [comment]
For the next release I'd change this to a list of dicts. Would make more sense.

**Pull request merged**